### ISSUE TITLE
Justification of accessibility measures

### ISSUE THREAD

**Issue by gregmacfarlane** [byu-transpolab/alameda_destinationchoice#14]
Reviewer B

> The paper should better differentiate the value of using alternative accessibility measures besides simplicity. One of the ongoing debates that cut-across transportation engineering, planning, and geography are how specific measures are more useful in planning or policy evaluation scenarios when observed behavior is not deemed adequate or valuable. This is particularly true when questions about distributive justice are paramount, and observed behavior only works to explain some inequalities but falls short in measuring un-meet travel demand needs. This is in addition to the ongoing concern by some scholars regarding the adoption of the concept in practice – see the most recent work on accessibility published in JAPA by Siddiq and Taylor, the work of Boisjoly and El-Geneidy on the same topic, and the work by A Paez titled ‘Measuring accessibility: positive and normative implementations of various accessibility indicators’ published in Transport Geography to start.

**Comment by gregmacfarlane** [comment]
We mention in the limitations why this might be true, but should probably bring this into the literature review.

**Comment by c-voulgaris** [comment]
Addressed in pull request. For response memo:

We have updated the literature review to discuss the referenced work comparing accessibility metrics in terms of both practicality/simplicity and suitability to equity evaluation.
